smt/applications/ego.py

```diff
@@ -17,7 +17,7 @@
 from smt.applications.mixed_integer import MixedIntegerContext
 from smt.utils.misc import compute_rms_error
 
-from smt.surrogate_models import KPLS, KRG, KPLSK
+from smt.surrogate_models import KPLS, KRG, KPLSK,MGP
```

Comment:
Nitpicking: would you mind passing black or adding a space after the comma?

Reply:
I applied black on the complete folder so other files as marked as changed

Reply:
Yes, I've seen that. It is ok.